addons/modules/functions/fnc_moduleHideTerrainObjects.sqf

```diff
@@ -0,0 +1,57 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to hide terrain objects.
+ *
+ * Arguments:
+ * 0: Logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call zen_modules_fnc_moduleHideTerrainObjects
+ *
+ * Public: No
+ */
+
+params ["_logic"];
+
+private _position = getPos _logic;
+deleteVehicle _logic;
+
+["STR_a3_to_hideTerrainObjects1", [
+    ["TOOLBOX:YESNO", "STR_a3_to_hideTerrainObjects1", true],
+    ["EDIT", ELSTRING(common,Range), 10],
+    ["CHECKBOX", "STR_ZEN_Modules_Buildings", true],
+    ["CHECKBOX", "STR_A3_CfgEditorCategories_EdCat_Walls0", true],
+    ["CHECKBOX", "STR_A3_CfgEditorSubcategories_EdSubcat_Plants0", true],
+    ["CHECKBOX", "STR_A3_CfgEditorSubcategories_EdSubcat_Default0", true]
```

Comment:
Let's use the same terminology as the 3DEN module. For example, it uses "Vegetation" instead of "Plants".

Reply:
97fdfbe